File: systems-for-design-and-implementation/labs/CSharp_ChatSockets/services/ChatException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace chat.services
{
    public class ChatException : Exception
    {
        public ChatException():base() { }

        public ChatException(String msg) : base(msg) { }

        public ChatException(String msg, Exception ex) : base(msg, ex) { }

    }
}

File: systems-for-design-and-implementation/labs/Java_ChatSockets/ChatServices/src/chat/services/ChatException.java
package chat.services;

/**
 * Created by IntelliJ IDEA.
 * User: grigo
 * Date: Mar 18, 2009
 * Time: 1:32:58 PM
 */
public class ChatException extends Exception
{
    public ChatException()
    {
    }

    public ChatException(String message)
    {
        super(message);
    }

    public ChatException(String message, Throwable cause)
    {
        super(message, cause);
    }
}
